Scoring.py

```diff
@@ -0,0 +1,34 @@
+import random, pygame, sys, pygame.font
```

Comment:
It's good practice to have a header comment at the top of every file to explain what this file does. That way when people are cruising through all the files they can easily read a summary of the code.